decidim-participatory_processes/app/cells/decidim/participatory_processes/process_s_cell.rb

```diff
@@ -3,7 +3,7 @@
 module Decidim
   module ParticipatoryProcesses
     # This cell renders the Search (:s) process card
-    # for an given instance of a ParticipatoryProcess
+    # for a given instance of a Process
```

Comment:
Since we are are referring to an instance of, i guess it should be the name of the class.
```suggestion
    # for a given instance of a ParticipatoryProcess
```